akka-docs/rst/java/serialization.rst

```diff
@@ -35,6 +35,14 @@ one of which all other candidates are superclasses. If this condition cannot be
 met, because e.g. ``java.io.Serializable`` and ``MyOwnSerializable`` both apply
 and neither is a subtype of the other, a warning will be issued.
 
+.. note::
+
+  If you are using Scala for your message protocol and your messages are contained
+  inside of a Scala wrapper object, then in order to reference those messages, you will need
+  use the de-sugared path. For a message named ``Message`` contained inside
+  the Scala object named ``Wrapper`` you would need to reference it as
+  ``Wrapper$Message`` instead of ``Wrapper.Message``
```

Comment:
Perhaps add a word of explanation, "This the how the class is represented once compiled by scalac, thus we need to use this path to reach it"?